orders/actions.py

```diff
@@ -25,6 +25,7 @@ def process_stripe_charge(order, token):
     except IntegrityError:
         stripe_integration.refund_charge(charge.id)
         mark_order_as_errored_after_charge(order, charge.id)
+        logger.exception('An order faced an integrity error')
```

Comment:
Do we have anything on the `IntegrityError` we could log for added help in resolving the problem?  If so, perhaps we could record it on the `Order` object.

Reply:
`logger.exception` logs the stack trace and reason for the integrity error - for example:
```django.db.utils.IntegrityError: duplicate key value violates unique constraint "tickets_ticketinvitation_email_addr_key"
DETAIL:  Key (email_addr)=(alice@example.com) already exists.```
I assume we've removed a significant source of potential `IntegrityError`s by making this change, and the logs appear in Heroku's dashboard, so therefore on the rare occasion they happen, it's probably just as easy to look at the logs.

Reply:
Of course, yes.